README.md

```diff
@@ -2,73 +2,55 @@
 
-Stratis Bitcoin
+Blockcore
 ===============
 
-https://stratisplatform.com
+https://blockcore.net
 
 Bitcoin Implementation in C#
 ----------------------------
 
-Stratis is an implementation of the Bitcoin protocol in C# on the [.NET Core](https://dotnet.github.io/) platform.  
-The node can run on the Bitcoin and Stratis networks.  
-Stratis Bitcoin is based on the [NBitcoin](https://github.com/MetacoSA/NBitcoin) project.  
+**What is Blockcore?**
 
-For Proof of Stake support on the Stratis token the node is using [NStratis](https://github.com/stratisproject/NStratis) which is a POS implementation of NBitcoin.  
+- Blockcore is a [.NET Core](https://dotnet.github.io/) platform to build decentralized networks based on the Bitcoin protocol
+- Blockcore is a platform to build Layer 1 consensus networks based on the Bitcoin protocol, built on the [.NET Core](https://dotnet.github.io/) framework and written entirely in C#. 
+- Blockcore aims to maintain an alternative C# Bitcoin implementation, based on the [NBitcoin](https://github.com/MetacoSA/NBitcoin) & [Stratis](https://github.com/stratisproject/StratisBitcoinFullNode) projects.
+- Blockcore isn't a coin.
+- Blockcore isn't a for profit business.
 
-[.NET Core](https://dotnet.github.io/) is an open source cross platform framework and enables the development of applications and services on Windows, macOS and Linux.  
+**Why Blockcore?**
 
-Join our community on [discord](https://discord.gg/9tDyfZs).  
+- We see a need within the crypto ecosystem for development of the C# full node technology.
+- Stratis has provided an excellent starting point but their focus is enterprise and businesses and we feel strongly that there is also value focusing on open and public blockchains.
 
-The design
-----------
+**Blockcore objectives**
 
-**A Modular Approach**
+- Continue development of the C# Stratis fullnode.
+- Maintain the C# Bitcoin fullnode.
+- Support projects and teams that use the underlying technology.
+- Extend the technology by building developer and user tools
+- Provide a forum for developers and teams to collaborate and improve on the technology.
 
-A Blockchain is made of many components, from a FullNode that validates blocks to a Simple Wallet that track addresses.
-The end goal is to develop a set of [Nuget](https://en.wikipedia.org/wiki/NuGet) packages from which an implementer can cherry pick what he needs.
+**Blockcore principles**
 
-* **NBitcoin**
-* **Stratis.Bitcoin.Core**  - The bare minimum to run a pruned node.
-* **Stratis.Bitcoin.Store** - Store and relay blocks to peers.
-* **Stratis.Bitcoin.MemoryPool** - Track pending transaction.
-* **Stratis.Bitcoin.Wallet** - Send and Receive coins
-* **Stratis.Bitcoin.Miner** - POS or POW
-* **Stratis.Bitcoin.Explorer**
+- We support public open blockchains, not working with enterprises or consulting with businesses  
```

Comment:
This should be changed, and there is nothing wrong with someone involved with blockcore, starting a business working with enterprise or consulting around blockcore. The blockcore open source ("team") itself won't do it, but we'll support any enterprise that is building solutions on blockcore, but that would be similar to any other open source contributor of community member.

Reply:
I wanted to be clear that we aren't a business & we aren't competing with the Stratis business model. But, I can see how that comes across as us discouraging people from doing that ... which I agree isn't the intention ... I'll re-word

Reply:
This I agree, we should not actually care who uses the platform, and if business want to use it there is an opportunity to have some premium offering (consulting, Dev work, architecture, code review)

We should aim to keep blockcore as a flexible platform as possible to enable extendebility

Reply:
In the latest update I have removed this statement entirely rather than reword, as in one of the other statements we say we support any project using the platform.